README.md

```diff
@@ -199,6 +199,7 @@ Assign a default value for each setting by passing environment variables to Libr
 | `USE_HLS`               | `["on", "off"]`                                                                                     | `off`         |
 | `HIDE_HLS_NOTIFICATION` | `["on", "off"]`                                                                                     | `off`         |
 | `AUTOPLAY_VIDEOS`       | `["on", "off"]`                                                                                     | `off`         |
+| `LINK_CONFIRMATION`     | `["on", "off"]`                                                                                     | `off`         |
```

Comment:
I would choose a different name for this option. `LINK_CONFIRMATION` is ambiguous; how about `VISIT_REDDIT_CONFIRMATION`?